sample_analyze_read.py

```diff
@@ -0,0 +1,192 @@
+    # Replace with your actual formUrl:
+    # If you use the URL of a public website, to find more URLs, please visit: https://aka.ms/more-URLs 
+    # If you analyze a document in Blob Storage, you need to generate Public SAS URL, please visit: https://aka.ms/create-sas-tokens
+    poller = document_intelligence_client.begin_analyze_document(
+        "prebuilt-read",
+        AnalyzeDocumentRequest(url_source=formUrl)
+    )       
+    # # If analyzing a local document, remove the comment markers (#) at the beginning of these 11 lines.
+    # # Delete or comment out the part of "Analyze a document at a URL" above.
+    # # Replace <path to your sample file>  with your actual file path.
+    # path_to_sample_document = "<path to your sample file>"
+    # with open(path_to_sample_document, "rb") as f:
+    #     poller = document_intelligence_client.begin_analyze_document(
+    #         "prebuilt-read",
+    #         analyze_request=f,
+    #         features=[DocumentAnalysisFeature.STYLE_FONT],
+    #         content_type="application/octet-stream",
+    #     )
+    result: AnalyzeResult = poller.result()
+    
+    # [START analyze_read]
+    # Detect languages.
+    print("----Languages detected in the document----")
+    if result.languages is not None:
+        for language in result.languages:
+            print(f"Language code: '{language.locale}' with confidence {language.confidence}")
+
+    # Detect styles.
+    print("----Styles detected in the document----")
+    if result.styles:
+        for style in result.styles:
+            if style.is_handwritten:
+                print("Found the following handwritten content: ")
+                print(",".join([result.content[span.offset : span.offset + span.length] for span in style.spans]))
+            if style.font_style:
+                print(f"The document contains '{style.font_style}' font style, applied to the following text: ")
+                print(",".join([result.content[span.offset : span.offset + span.length] for span in style.spans]))
+    
+    # To learn the detailed concept of "bounding polygon" in the following content, visit: https://aka.ms/bounding-region
+    # Analyze pages.
+    for page in result.pages:
+        print(f"----Analyzing document from page #{page.page_number}----")
+        print(f"Page has width: {page.width} and height: {page.height}, measured with unit: {page.unit}")
+
+        # Analyze lines.
+        if page.lines:
+            for line_idx, line in enumerate(page.lines):
+                words = get_words(page, line)
+                print(
+                    f"...Line # {line_idx} has {len(words)} words and text '{line.content}' within bounding polygon '{line.polygon}'"
+                )
+
+                # Analyze words.
+                for word in words:
+                    print(f"......Word '{word.content}' has a confidence of {word.confidence}")
+
+        # Analyze selection marks.
```

Comment:
Read model should not have selection marks. It's only available in Layout and some prebuilt model.

Reply:
Already deleted